### ISSUE TITLE
Email alert for overdue license

### ISSUE THREAD

**Issue by chekwan** [grokability/snipe-it#7080]
#### Expected Behavior (or desired behavior if a feature request)

I just found out that theres no email notification for Overdue licensed. It only for asset but not for licensed.

please assist

-----

#### Actual Behavior

(what actually happens goes here)

-----

#### Please confirm you have done the following before posting your bug report:

- [ ] I have enabled debug mode 
- [ ] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

-----
#### Provide answers to these questions:

- Is this a fresh install or an upgrade? 
Fresh Install

- Version of Snipe-IT you're running
v4.6.6

- Version of PHP you're running
 5.6.24

- Version of MySQL/MariaDB you're running
- What OS and web server you're running Snipe-IT on
Linux

- What method you used to install Snipe-IT (install.sh, manual installation, docker, etc)
- WITH DEBUG TURNED ON, if you're getting an error in your browser, include that error
- What specific Snipe-IT page you're on, and what specific element you're interacting with to trigger the error
- If a stacktrace is provided in the error, include that too.
- Any errors that appear in your browser's error console.
- Confirm whether the error is reproducible on the demo: https://snipeitapp.com/demo.
- Include any additional information you can find in `storage/logs` and your webserver's logs.
- Include what you've done so far in the installation, and if you got any error messages along the way.
- Indicate whether or not you've manually edited any data directly in the database

Please do not post an issue without answering the related questions above. If you have opened a different issue and already answered these questions, answer them again, once for every ticket. It will be next to impossible for us to help you.

https://snipe-it.readme.io/docs/getting-help

**Comment by eandry77** [comment]
Funny, I was just about to open a ticket for the same request !
I need this feature too !

**Comment by snipe** [comment]
What exactly does this mean? There is no expected checkin for licenses, so what would make it overdue, exactly? `php artisan snipeit:expiring-alerts` includes licenses.

**Comment by eandry77** [comment]
The expiring-alerts sends mails for licences which are about to expire, but once expired, no alerts anymore. Maybe adding the expired licenses to snipeit:expiring-alerts, or create snipeit:expired-alerts ?

**Comment by snipe** [comment]
That’s the intended behavior. If we’ve been warning you every day for 60 days that your licenses are expiring, why would we think you’re still going to want to hear about it on day 31? The expired licensees list would grow quite long (and probably irrelevant) for most organizations.

**Comment by chekwan** [comment]
Hi Snipe,

Means the email alert only the days for licensed to expired, and don't have for overdue

What if the licensed expired is overdue (user not renew the licensed yet).?
Any email alert for this situation.?

**Comment by chekwan** [comment]
> 
> 
> What exactly does this mean? There is no expected checkin for licenses, so what would make it overdue, exactly? `php artisan snipeit:expiring-alerts` includes licenses.

How about for the Software Licensed is already expired and user forgot to renew. 
Can the system send the email alert to the user.?

**Comment by snipe** [comment]
> Means the email alert only the days for licensed to expired, and don't have for overdue

Right, because we've already been emailing you for weeks about the upcoming expiration. 

> Can the system send the email alert to the user especially how many days the licensed overdue from the expiry date.?

Most users aren't in charge of renewing their own licenses though. We'd be generating a lot of email to people who have no control over the solution.

**Comment by chekwan** [comment]
Any advice if I'm still want the email alerts for overdue expired licensed.?

And the alert will send the email notification same flow as the Expiring Alerts Threshold to the Email addresses or distribution lists you want alerts to be sent to using the cronjob.
It only send email notification that contained:
- licensed name
- person in charged
- expired date
- overdue days

**Comment by chekwan** [comment]
Hi Snipe.

Any comment.?

What if I pay u a little bit to do some script  for this request.?

**Comment by snipe** [comment]
Also requested via [FD-49345]

**Comment by snipe** [comment]
This was solved some time ago - `snipeit:expiring-alerts --expired-licenses` will include licenses that are already expired.
